docs/source/ecf.rst

```diff
@@ -269,6 +269,10 @@ photometry
 ''''''''''
 Only used for photometry analyses. Set to True if the user wants to analyze a photometric dataset.
 
+convert_to_e
```

Comment:
This isn't a required setting (by default is set to true if not present in the ECF) and is only needed if you want to do flux-calibrated photometry. I can add it to the NIRCam and MIRI photometry ECFs if you want, but I really don't think it should be added to the spectroscopy ECFs since it's likely not overly useful there and more likely to cause confusion than anything else

Reply:
Please add it to the spectroscopic ECFs.  Today, I've been trying to generate a calibrated stellar spectrum with G395 and has to comment out the conversion line in Stage 3.